src/network/serveropcodes.cpp

```diff
@@ -145,70 +145,70 @@ const ClientCommandFactory clientCommandFactoryTable[TOCLIENT_NUM_MSG_TYPES] =
 	null_command_factory,
 	null_command_factory,
 	null_command_factory,
-	{ "TOCLIENT_BLOCKDATA",                2, true }, // 0x20
-	{ "TOCLIENT_ADDNODE",                  0, true }, // 0x21
-	{ "TOCLIENT_REMOVENODE",               0, true }, // 0x22
+	{ "TOCLIENT_BLOCKDATA",                     2, true }, // 0x20
+	{ "TOCLIENT_ADDNODE",                       0, true }, // 0x21
+	{ "TOCLIENT_REMOVENODE",                    0, true }, // 0x22
 	null_command_factory,
 	null_command_factory,
 	null_command_factory,
 	null_command_factory,
-	{ "TOCLIENT_INVENTORY",                0, true }, // 0x27
+	{ "TOCLIENT_INVENTORY",                     0, true }, // 0x27
 	null_command_factory,
-	{ "TOCLIENT_TIME_OF_DAY",              0, true }, // 0x29
-	{ "TOCLIENT_CSM_RESTRICTION_FLAGS",    0, true }, // 0x2A
+	{ "TOCLIENT_TIME_OF_DAY",                   0, true }, // 0x29
+	{ "TOCLIENT_CSM_RESTRICTION_FLAGS",         0, true }, // 0x2A
+	{ "TOCLIENT_CAMERA_MODE_RESTRICTION_FLAGS", 0, true }, // 0x2B
```

Comment:
`_MODE_` could be removed, maybe just make it TOCLIENT_CAMERA_RESTRICTION_FLAGS ? Not that important though.

Reply:
I felt `mode` made it clear that this restricts camera modes, and not the camera itself (i.e. camera movement, rotation, etc.)

Reply:
Should I omit `_MODE_` and if yes, should I omit it everywhere (incl. packet name, getter/setter names, Lua API method name, etc.)?

Reply:
What about `TOCLIENT_CAMERA_MODE_FLAGS`? Imho that the flags restrict seems kinda obvious.

Reply:
`TOCLIENT_CAMERA_MODES` based on my yesterday's suggested design idea.

Reply:
Yes, `TOCLIENT_CAMERA_MODES` is better suited for the name, due to SmallJoker's idea.

Reply:
I thought of `TOCLIENT_CAMERA_MODES` as well, and that is also better suited for the packet name, due to [SmallJoker's idea](https://github.com/minetest/minetest/pull/8409#discussion_r270634673).

Reply:
Shortened the name across the PR to camera modes. Packet renamed to `TOCLIENT_CAMERA_MODES` (yay!)